### PR TITLE
dark mode and UI fix

### LINKED ISSUE #497: It should be great to add dark mode in docs
**Is your feature request related to a problem? Please describe.**
We all are developers and we a all know that we love dark modes. So that, bugs and hide 🤣.

**Describe the solution you'd like**
It would be great to add dark mode in docs.

**Describe alternatives you've considered**
I think docs are built with docusaurus. it can be a easy change to do.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by priyanshuverma-dev**
closes #497 

## 📑 Description
Adds dark mode in Docs and UI tweaks
## ✅ Checks
- [ ] I have updated the documentation as required
- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by priyanshuverma-dev** [comment]
@petar-cvit PTAL!

File: web/docusaurus.config.js [hunk 1]
```diff
@@ -82,34 +82,14 @@ const config = {
...
-          {
-            href: "https://x.com/CyclopsUI",
-            position: "right",
-            className: "header-x-link",
-          },
-          {
-            href: "https://discord.com/invite/8ErnK3qDb3",
-            className: "header-discord-link",
-            position: "right",
-          },
```
Comment (petar-cvit) [inline web/docusaurus.config.js:112]: We need the links to our socials, so we can't remove them. You can add the right padding to `header-discord-link` class to split the theme switch icon from the socials

File: web/docusaurus.config.js [hunk 2]
```diff
@@ -82,34 +82,14 @@ const config = {
             type: "html",
             position: "right",
             value:
-              '<a href="/docs/installation/install/manifest" style="background-color: #FF8803; color: #FFF; height: 30px; width: 150px; text-decoration: none; display: inline-flex; align-items: center; border-radius: 30px 30px 30px 30px"' +
+              '<a href="/docs/installation/install/manifest" style="background-color: #FF8803; color: #FFF; height: 30px; width: 150px; text-decoration: none; display: flex; align-content: center; justify-content: center; border-radius: 30px 30px 30px 30px"' +
               "onmouseover=\"this.style.backgroundColor='#FFA229';\"" +
               "onmouseout=\"this.style.backgroundColor='#FF8803';\">" +
-              '<h3 style="margin: 0px auto;">' +
+              "<h3>" +
```
Comment (petar-cvit) [inline web/docusaurus.config.js:88]: Can you revert this? Its not centered with this css

File: web/src/css/custom.css [hunk 3]
```diff
@@ -6,18 +6,22 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #fe8801;
+  --ifm-color-primary: #f39422;
```
Comment (petar-cvit) [inline web/src/css/custom.css:9]: Can you revert this. We have the same orange hex on all the places

File: web/src/css/custom.css [hunk 4]
```diff
@@ -105,16 +109,16 @@
...
+  background-color: #0a1120;
 }
 
 .card {
   background-color: #fff;
 }
 
 .navbar {
-  background-color: rgba(0, 8, 48, 0.7);
+  background-color: rgba(0, 11, 20, 0.7);
```
Comment (petar-cvit) [inline web/src/css/custom.css:121]: I'd leave the navbar as it was

File: web/src/css/custom.css [hunk 5]
```diff
@@ -105,16 +109,16 @@
     no-repeat;
 }
 
-.docusaurus-mt-lg {
-  background-color: #fff;
+[data-theme="dark"] .card {
+  background-color: #0a1120;
```
Comment (petar-cvit) [inline web/src/css/custom.css:113]: What is affected by this?

**Review by petar-cvit — CHANGES_REQUESTED** [review]
@priyanshuverma-dev, the dark mode looks awesome! Left some comments

File: web/docusaurus.config.js (hunk 1 above)
Comment (priyanshuverma-dev) [reply web/docusaurus.config.js:112]: can we change the icons style (svg)? @petar-cvit .

File: web/docusaurus.config.js (hunk 2 above)
Comment (priyanshuverma-dev) [reply web/docusaurus.config.js:88]: reverted 👍

File: web/docusaurus.config.js (hunk 1 above)
Comment (petar-cvit) [reply web/docusaurus.config.js:112]: How would you change them?

File: web/src/css/custom.css (hunk 5 above)
Comment (priyanshuverma-dev) [reply web/src/css/custom.css:113]: It changes the pagination card colors @petar-cvit .

File: web/docusaurus.config.js (hunk 1 above)
Comment (priyanshuverma-dev) [reply web/docusaurus.config.js:112]: I can change by adding different svgs by setting type to html

Comment (priyanshuverma-dev) [reply web/docusaurus.config.js:112]: It look like this now:
image

File: web/src/css/custom.css [hunk 6]
```diff
@@ -92,27 +94,63 @@
...
+  margin: 0px;
+  height: 30px;
+  width: 30px;
+  padding: 0px;
+  border-radius: 50%;
+}
+
+.link:hover {
+  background-color: #444950;
+  transition: background var(--ifm-transition-fast);
```
[the remark this answers is not in the document]
Comment (priyanshuverma-dev) [reply web/src/css/custom.css:135]: when we hover the div background colour and transition is copied as it is of theme switch btn

File: web/src/css/custom.css [hunk 7]
```diff
@@ -12,12 +12,16 @@
 }
 
 .footer {
-  background-color: #000830;
+  background-color: #242526;
+}
+
+[data-theme="dark"] .footer {
+  background-color: #080e1b;
```
[the remark this answers is not in the document]
Comment (priyanshuverma-dev) [reply web/src/css/custom.css:19]: these are little bit diff
```css
.footer {
  background-color: #242526;
}

[data-theme="dark"] .footer {
  background-color: #080e1b;
}
```
Should i change it same

**Comment by priyanshuverma-dev** [comment]
@petar-cvit A gentle reminder to review.

**Review by petar-cvit — APPROVED** [review]
Thanks @priyanshuverma-dev 🧡

**Pull request merged**